src/runtime_src/core/common/xclbin_parser.h

```diff
@@ -424,6 +424,15 @@ get_project_name(const axlf* top);
 std::string
 get_fpga_device_name(const char* xml_data, size_t xml_size);
 
+kernel_properties::mailbox_type
+get_mailbox_from_ini(const std::string& kname);
+
+kernel_properties::restart_type
+get_restart_from_ini(const std::string& kname);
+
+bool
+get_sw_reset_from_ini(const std::string& kname);
+
```

Comment:
Still open?

Reply:
I need these functions while creating properties in ELF flow as well @stsoe , should I use static functions there as well? I moved these functions in header so that there wont be code duplicity